runtime/altair/src/evm.rs

```diff
@@ -91,6 +90,6 @@ impl pallet_ethereum_transaction::Config for Runtime {
 }
 
 impl axelar_gateway_precompile::Config for Runtime {
-	type AdminOrigin = EnsureRoot<AccountId>;
+	type AdminOrigin = EnsureRootOr<HalfOfCouncil>;
```

Comment:
Out of curiosity: Do want to propose limiting to `Root` long-term?

Reply:
I guess gov v2 will anyways make Council obsolete, right?